CodeStarterCamp_Week1/main.swift

```diff
@@ -8,23 +8,29 @@
 
 import Foundation
 
-func printIceBar(rows: Int) {
-    for _ in 1...rows {
-        var star = ""
-        for _ in 1...11 {
-            star = star + "*"
-            //star += "*"
-        }
-        print(star)
+func printPeperoInfo(bodyLength: Int, topping: String, stickLength: Int) {
+    print("<정보>", "길이: \(bodyLength)", "몸통: \(peperoBody)", "토핑: \(topping)", "막대길이: \(stickLength)", separator: "\n")
+    print()
+}
+
+func drawPeperoStick(stickLength: Int) {
+    let PeperoStick = " | | "
+    for _ in 1...stickLength {
+        print(PeperoStick)
     }
 }
 
-func printIceStick() {
-    let bar = "    | |    "
-    for _ in 1...4 {
-        print(bar)
+func drawPeperoWithTopping(bodyLength: Int, topping: String) {
+    for _ in 1...bodyLength {
+        print(topping + peperoBody + topping)
     }
 }
 
-printIceBar(rows: 8)
-printIceStick()
+let bodyLength = 6
+let peperoBody = "|0|"
+let topping = " "
+let stickLength = 4
+
+printPeperoInfo(bodyLength: bodyLength, topping: topping, stickLength: stickLength)
+drawPeperoWithTopping(bodyLength: bodyLength, topping: topping)
+drawPeperoStick(stickLength: stickLength)
```

Comment:
```suggestion
printPeperoInfo(bodyLength: 6, topping: " ", stickLength: 4)
drawPeperoWithTopping(bodyLength: 6, topping: " ")
drawPeperoStick(stickLength: 4)
```

불필요한 변수는 없어도 될 거 같습니다. 
그리고 빼빼로 몸통에 관한 매개변수가 없는 거 같아요! (그럼에도 정상 실행된 이유는 `peperoBody`가 전역변수라서 그런 거 같네용)